src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/Configuration.php

```diff
@@ -72,6 +72,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('slugifier')
                     ->defaultValue('Sulu\Bundle\DocumentManagerBundle\Slugifier\Urlizer::urlize')
+                    ->setDeprecated()
```

Comment:
Is just the `defaultValue` deprecated, or the entire option? Is the `slugifier` option used at all now?

Reply:
The whole option. It's not used. Internally now it uses PathCleaner.

Reply:
I think the `setDeprecated` function takes a message as argument, right? Then I would add one to make it more clear:

```
The "sulu_document_manager.slugifier" configuration option is not used anymore.
```

Reply:
added better messages. As you said @internal says what it have to say, no need for explanatory. Here is the same. :)